projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts

```diff
@@ -53,149 +79,150 @@ context('CPQ Configuration', () => {
         'GET',
         `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
           'BASE_SITE'
-        )}/products/suggestions?term=${testProduct}*`
+        )}/products/suggestions?term=${PROD_CODE_CAM}*`
       );
-      productSearch.searchForProduct(testProduct);
+      productSearch.searchForProduct(PROD_CODE_CAM);
       configuration.clickOnConfigureBtnInCatalog();
     });
 
     it('should be able to navigate from the product details page', () => {
-      configuration.goToPDPage(powertoolsShop, testProduct);
+      configuration.goToPDPage(POWERTOOLS, PROD_CODE_CAM);
       configuration.clickOnConfigureBtnInCatalog();
     });
   });
 
-  describe('Update attribute values', () => {
-    it('should support update values for radio button attribute type', () => {
-      configuration.goToPDPage(powertoolsShop, testProductCoffeeMachine);
+  describe('UiTypes', () => {
```

Comment:
Could you write longer description? For instance: **Handling different UI types**